releasenotes/notes/deprecation-removals-3d445f09f3fa0044.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    Passing the ``carriers`` argument to ``InstructionToSignals`` as a ``list`` is no longer
+    supported.
+  - The ``Solver.signals`` property, and :class:`.Solver` init arguments ``hamiltonian_signals``
```

Comment:
this line is a bit confusing to read. Maybe just split into two lines

Reply:
Broke it into separate lines. Also I realized that I didn't use proper formatting for the bullet points. Fixed each one to have format:

```
- |
   This is the bullet text.
```